nvi-evaluator/src/main/java/no/sikt/nva/nvi/evaluator/calculator/CandidateCalculator.java

```diff
@@ -182,24 +197,31 @@ private List<Affiliation> filterNviAffiliations(Contributor contributor) {
         return contributor.affiliations()
                    .stream()
                    .map(Affiliation::id)
+                   .map(this::fetchOrganization)
+                   .map(Organization::getTopLevelOrg)
+                   .map(Organization::id)
                    .map(URI::toString)
                    .filter(this::isNviInstitution)
                    .map(URI::create)
                    .map(Affiliation::new)
                    .toList();
     }
 
-    private boolean isNviInstitution(String affiliation) {
-        var response = getResponse(affiliation);
+    private Organization toCristinOrganization(String response) {
+        return attempt(() -> dtoObjectMapper.readValue(response, Organization.class)).orElseThrow();
+    }
+
+    private boolean isNviInstitution(String institutionId) {
+        var response = getResponse(createCustomerApiUri(institutionId));
         if (isSuccessOrNotFound(response)) {
-            LOGGER.info(AFFILIATION_FETCHED_SUCCESSFULLY_MESSAGE, response.statusCode());
+            LOGGER.info(CUSTOMER_FETCHED_SUCCESSFULLY_MESSAGE, response.statusCode());
```

Comment:
Do we need to log it?

Reply:
No, maybe not on successful operation. Removing